Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu
+
+# Update the apt repository
+RUN apt update
+
+# Install dependencies
+RUN apt-get install -y build-essential
```

Comment:
I know performance is probably not needed but maybe to keep it to a single layer you could do

```
RUN apt-getinstall -y build-essential file zsh git sudo ruby curl vim language-pack-en

Reply:
Fixed!